src/Google/Ads/GoogleAds/Lib/V10/GoogleAdsCallLogger.php

```diff
@@ -35,6 +35,10 @@ final class GoogleAdsCallLogger
     private $endpoint;
     private $logMessageFormatter;
     private $context;
+    // An associative array of all the log levels (based on the PSR-7 standard) to sequence numbers.
```

Comment:
I think you mean the PSR-3 standard?

https://www.php-fig.org/psr/psr-3/

Reply:
Sorry. Done.